erts/emulator/test/float_SUITE_data/fp_drv.c

```diff
@@ -123,7 +123,7 @@ static ErlDrvSSizeT control(ErlDrvData drv_data,
 	    res_str = "skip: no thread support";
 	else if (0 != erl_drv_thread_create("test", &tid, do_test, NULL, NULL))
 	    res_str = "failed to create thread";
-	else if (0 != erl_drv_thread_join(tid, &res_str))
+	else if (0 != erl_drv_thread_join(tid, NULL))
```

Comment:
This causes the test case to fail. As the exit value is a `char*`, cast the result pointer to `void**` instead of ignoring it.

```suggestion
	else if (0 != erl_drv_thread_join(tid, (void**)&res_str))
```

Reply:
Alright. I'll change it. Thanks.